src/LettuceEncrypt/Acme/ICertificateAuthorityConfiguration.cs

```diff
@@ -12,4 +12,11 @@ public interface ICertificateAuthorityConfiguration
     /// The base uri of the ACME protocol.
     /// </summary>
     Uri AcmeDirectoryUri { get; }
+
+    /// <summary>
```

Comment:
What is the expected format of the additional issuers? PEM? DER? 

Also, would it be feasible to add some test coverage to assert this is working correctly?

Reply:
Addressed in [b02bce6](https://github.com/natemcmaster/LettuceEncrypt/pull/281/commits/b02bce6b2118c6358b98c36b557828c7aca5cc5a) and [daacdba](https://github.com/natemcmaster/LettuceEncrypt/pull/281/commits/daacdba2eb7747380d6d1371493225391076e76d)

Testing wasn't that easy, as the code that needs testing is directly bound to implementations from certes. I've added a (internal only!) abstraction to make it testable, and extracted (and exposed internally) the part to be tested into it's own method without changing the control flow of the existing code. Hope that's okay with you! I __think__ no more tests are needed, but please let me know if you feel otherwise :)